src/site/content/en/lighthouse-pwa/service-worker/index.md

```diff
@@ -1,53 +1,56 @@
 ---
 layout: post
-title: Registers a service worker that controls page and start_url
+title: "Does not register a service worker that controls page and `start_url`"
```

Comment:
```suggestion
title: Does not register a service worker that controls page and start_url
```

Looks like this was copy-pasted from the audit source and might still have some formatting left over from that

Reply:
Actually, web.dev supports code font in titles now. That should be retained.

Reply:
Oh, you didn't commit this before you merged. All good.

Reply:
Were the double quotes intentional too? We don't seem to do that in any of the other titles.

Reply:
I actually meant to commit it but just missed this one :D maybe the GitHub UI hid it...

Reply:
Yes, the double quotes are required when there's MD in the title.